docs/source/customize.rst

```diff
@@ -77,6 +78,13 @@ name of a plain text file containing one word per line. All of the
 words in the file are assumed to be spelled correctly and may appear
 in any part of the document being processed.
 
+You can use multiple text files with words to be added to the dictionary,
+to do this all you need to do is use a list and include the name of your
+text files.
+
+Example:
```

Comment:
If you use 2 colons to end line 85 and then add a blank line after, the example will be formatted as code. I will take care of that in a follow-up patch to save you the trouble this time.